package.json

```diff
@@ -48,6 +48,7 @@
     "chalk": "^1.0.0",
     "connect": "2.8.3",
     "debug": "~2.1.0",
+    "flow-bin": "^0.11.0",
```

Comment:
This should be "0.11.0" (or I guess 0.13.1 now) without the caret since .flowconfig's [version] doesn't support semver yet (I think?). Opened up https://github.com/facebook/flow/issues/592.